site.yml

```diff
@@ -201,24 +243,10 @@
 
     - name: Create jenkins backup job
       cron: name="Backup Jenkins" minute=0 hour=4
-            user="root" job="s3cmd -c /home/{{ remote_user }}/.s3cfg put -r /var/lib/jenkins/backups/*.zip s3://{{ aws_backup_bucket }} --encrypt 2> /dev/null"
+            user="root"
+            job="(s3cmd -c /home/{{ remote_user }}/.s3cfg put -r /var/lib/jenkins/backups/*.zip s3://{{ aws_backup_bucket }} --encrypt 2> /dev/null) && curl -s  {{ jenkins_backup_dms }}"
```

Comment:
are AWS creds stored on disk? We've been using the aws cli w/ the [AWS metadata service](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#instance-metadata-security-credentials) so creds aren't stored locally. Probably fine as-is though.